examples/base/params.py

```diff
@@ -109,6 +93,46 @@ class MyDecoratedFrame:
 
 # %%
 
-mypf.update_values({"A": {"value": 6, "unit": ""}})
+mypf.update_values({"R_0": {"value": 6000, "unit": "mm"}})
 
 print(mypf)
+
+# %%[markdown]
+
+# Accessing the value of a parameter in a different unit
+
+# %%
+
+print(mypf.R_0.value_as("cm"))  # 600
+
+# %%[markdown]
+# ## Raw conversion
+
+# In some situations it may be useful to convert a raw value to a different unit.
+# It is recommended (and enforced within bluemira core) that all conversions however
+# simplistic are converted this way.
+#
+# Any possible future base unit changes can be achieved by searching for a given unit.
```

Comment:
I'm not entirely sure what you mean here.

Reply:
Reworded in units.py